nodedata/nodedata.go

```diff
@@ -438,7 +441,7 @@ func (c *NodeData) checkAll() bool {
 
 			// Subscribe will set lastCheck to min value and you expect update in such a case
 			ignoreCache := toBeCheckedBy.Year() <= 1
-			resp, hash, err := c.checkOne(s.identifier, s.getAPI, s.settings, s.hash, ignoreCache, reportAnyway, reportNodeAnyway)
+			resp, hash, err := c.checkOne(s.identifier, s.getAPI, s.settings, s.hash, ignoreCache, reportAnyway, reportNodeAnyway, &s.lastNotSyncedToChain)
```

Comment:
usually is much easier to follow non negated variables because you need less logic in brains.

`lastSyncedToChain` >>> `lastNotSyncedToChain`